crates/solver/src/arguments.rs

```diff
@@ -417,6 +417,7 @@ impl std::fmt::Display for Arguments {
             "token_list_restriction_for_price_checks: {:?}",
             self.token_list_restriction_for_price_checks
         )?;
+        writeln!(f, "{}", self.s3_upload)?;
```

Comment:
Small nit: All the other argument structs that get flattened into this struct are at the top of this function. Makes some sense to keep them grouped together.